src/app/containers/StoryPromo/IndexAlsos/index.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { useContext, Fragment } from 'react';
+import { oneOfType, arrayOf, shape, number, string } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import pathOr from 'ramda/src/pathOr';
+import MediaIndicator from '@bbc/psammead-media-indicator';
+import {
+  IndexAlsos,
+  IndexAlso,
+  IndexAlsosUl,
+  IndexAlsosLi,
+} from '@bbc/psammead-story-promo/index-alsos';
+import { ServiceContext } from '../../../contexts/ServiceContext';
+
+const IndexAlsosContainer = ({ alsoItems, script, service }) => {
+  const {
+    translations: { media: mediaTranslations },
+  } = useContext(ServiceContext);
+
+  const getMediaType = (cpsType, mediaType) => {
+    const isPGL = cpsType === 'PGL';
+    const isMedia = cpsType === 'MAP';
+    const media = mediaType || 'Video';
+
+    if (!isPGL && !isMedia) {
+      return null;
+    }
+
+    const type = isPGL ? 'photogallery' : media.toLowerCase();
+
+    return type;
+  };
+
+  const IndexAlsosMediaIndicator = (cpsType, mediaType) => {
```

Comment:
this is a function, not a component, so should start with a lower case character
```suggestion
  const buildIndexAlsosMediaIndicator = (cpsType, mediaType) => {
```